src/lib/builders/tooltip/create.ts

```diff
@@ -40,11 +40,15 @@ const defaults = {
 	portal: 'body',
 	closeOnEscape: true,
 	disableHoverableContent: false,
+	group: undefined,
 } satisfies CreateTooltipProps;
 
 type TooltipParts = 'trigger' | 'content' | 'arrow';
 const { name } = createElHelpers<TooltipParts>('tooltip');
 
+// Store a global map to get the currently open tooltip.
+const _groupMap = new Map<string | boolean, Writable<boolean>>();
```

Comment:
Probably should be `string | true`, as we don't want `false` to be a valid group option

Reply:
Also, why the underline?

Reply:
I did handle the `false` case below, where it acts similar to `undefined`. I think it's a valid choice as both as "falsy", but I can remove it if you like.

The underline, eh, idk? It's internal global state, I just did it out of habit. I'll remove it.